packages/jaeger-ui-components/src/TracePageHeader/TracePageHeader.test.tsx

```diff
@@ -14,25 +14,34 @@
 
 import { render, screen } from '@testing-library/react';
 import React from 'react';
+import { ViewRangeTimeUpdate } from 'src/TraceTimelineViewer/types';
 
 import traceGenerator from '../demo/trace-generators';
 import { getTraceName } from '../model/trace-viewer';
 import transformTraceData from '../model/transform-trace-data';
 
-import TracePageHeader from './TracePageHeader';
+import TracePageHeader, { TracePageHeaderEmbedProps } from './TracePageHeader';
 
 const trace = transformTraceData(traceGenerator.trace({}));
-const setup = (propOverrides) => {
+const setup = (propOverrides?: TracePageHeaderEmbedProps) => {
   const defaultProps = {
+    canCollapse: false,
+    hideSummary: false,
+    onSlimViewClicked: () => {},
+    onTraceGraphViewClicked: () => {},
+    slimView: false,
     trace,
     hideMap: false,
     showArchiveButton: false,
     showShortcutsHelp: false,
     showStandaloneLink: false,
     showViewOptions: false,
     textFilter: '',
-    viewRange: { time: { current: [10, 20] } },
+    timeZone: '',
+    viewRange: { time: { current: [10, 20] as [number, number] } },
     updateTextFilter: () => {},
```

Comment:
Also remove this

Reply:
https://github.com/grafana/grafana/pull/59256/commits/9635fbe84940125adc4a5c6b30c0a94facfdb28d